e-commerce-project/src/app/components/login-page/login-page.component.ts

```diff
@@ -7,13 +8,20 @@ import { Component, OnInit } from '@angular/core';
 })
 export class LoginPageComponent implements OnInit {
 
-  toSubmit(userName, password){
-    console.log("Username is '" + userName + "' and password is '" + password + "'.");
-  }
+  loginForm: FormGroup;
 
-  constructor() { }
+  constructor(public formBuilder: FormBuilder) { 
+  	this.loginForm = formBuilder.group({
+  		userName: [null],
```

Comment:
`userName` we have only first argument in the array which stands for initial / start value, here we defined as null